control/tests/statesp_test.py

```diff
@@ -561,6 +561,69 @@ def test_str(self):
         assert str(sysdt1) == tref + "\ndt = 1.0\n"
 
 
+    def test_latex_repr(self):
```

Comment:
In the spirit of #438:

There are 4 identical tests with parameter variations. I suggest you use something like

```python
    @pytest.mark.parametrize(
        "g, repr_latex, p3ref, p5ref",
        [(g1, None,          g1_p3_p, g1_p5_p),
         (g1, "partitioned", g1_p3_p, g1_p5_p),
         (g1, "separate",    g1_p3_s, g1_p5_s),
         (g2, "partitioned", g2_p3_p, g2_p5_p),
         (g2, "separate",    g2_p3_s, g2_p5_s)])
    def test_latex_repr(self, g, repr_latex, p3ref, p5ref):
        try:
            # add "editsdefaults" fixture and remove the reset_defaults as soon as #438 is merged
            reset_defaults()
            if repr_latex is not None:
                set_defaults('statesp', latex_repr_type=repr_latex)
            assert g1._repr_latex_() == p3ref
            set_defaults('statesp', latex_num_format='.5g')
            assert g1._repr_latex_() == p5ref
        finally:
            reset_defaults()
```

Reply:
OK, looks good. Would this need to wait for #438, or do we already have pytest as a dependency?

Reply:
Except for the `editsdefaults` part, which is introduced by that PR, you should be able to do it right away. Pytest is already used and required on other tests.